staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/metrics.go

```diff
@@ -295,13 +295,11 @@ var (
 	)
 	apiserverWorkEstimatedSeats = compbasemetrics.NewHistogramVec(
 		&compbasemetrics.HistogramOpts{
-			Namespace: namespace,
-			Subsystem: subsystem,
-			Name:      "work_estimated_seats",
-			Help:      "Number of estimated seats (maximum of initial and final seats) associated with requests in API Priority and Fairness",
-			// the upper bound comes from the maximum number of seats a request
-			// can occupy which is currently set at 10.
-			Buckets:        []float64{1, 2, 4, 10},
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "work_estimated_seats_ratio",
+			Help:           "Ratio of estimated and maximum number of seats associated with requests in API Priority and Fairness",
```

Comment:
s/estimated and maximum of/maximum (of initial and final) divided by configured bound on/

Reply:
Changed.